src/mango/test/20-no-timeout-test.py

```diff
@@ -18,15 +18,22 @@
 class LongRunningMangoTest(mango.DbPerClass):
     def setUp(self):
         self.db.recreate()
+        self.db.create_index(["value"])
         docs = []
         for i in range(100000):
-            docs.append({"_id": str(i), "another": "field"})
-            if i % 20000 == 0:
+            docs.append({"_id": str(i), "another": "field", "value": i})
+            if i % 1000 == 0:
                 self.db.save_docs(docs)
                 docs = []
 
     # This test should run to completion and not timeout
     def test_query_does_not_time_out(self):
```

Comment:
This test is a bit pointless now that we're not doing sharded view builds as this was due to the timeout in the coordinator not hearing from shards that had a selective coordinator. You removed the whole RPC worker ping logic so I'd probably just delete this test since its not much more than testing the logic on waiting for a large view which exists elsewhere.